### DISCUSSION TITLE
My Webinar on R Markdown v1 at the ASA Philadelphia Chapter | /en/2023/05/philly-markdown/

### DISCUSSION THREAD

**Discussion by giscus** [rbind/yihui#1487] [Comments]
Tomorrow I&rsquo;ll give a webinar at 10am (US Central Time) at the ASA Philadelphia Chapter on R Markdown v1. If anyone is interested, you may register for the webinar here.
I have done a similar &hellip;

https://yihui.org/en/2023/05/philly-markdown/

**Comment by TimTaylor** [comment]
This is great! Do you know if a recording of the latest talk will be made available?

**Reply by yihui** [reply]
Sorry, it was not recorded, but I think you can just read the slides and won't really miss much information.
